include/generators/order_polytope_generator.h

```diff
@@ -0,0 +1,48 @@
+#include <sstream>
+#include <unordered_map>
+#include "misc.h"
+#include "misc/poset.h"
+
+
+// Instances taken from: https://github.com/ttalvitie/le-counting-practice
+static const std::unordered_map<std::string, std::string> instances =
+{
+    {"bipartite_0.5_008_0", R"(0 0 0 0 1 0 1 0
+                               0 0 0 0 1 0 0 0
+                               0 0 0 0 1 1 0 1
+                               0 0 0 0 1 0 1 0
+                               0 0 0 0 0 0 0 0
+                               0 0 0 0 0 0 0 0
+                               0 0 0 0 0 0 0 0
+                               0 0 0 0 0 0 0 0)"},
+
+    {"bayesiannetwork_andes_008_0", R"(0 0 0 0 0 0 0 0
+                                       1 0 0 0 0 0 0 0
+                                       0 1 0 0 0 0 0 0
+                                       0 0 0 0 0 0 0 0
+                                       0 0 1 1 0 0 0 0
+                                       0 0 0 0 1 0 0 0
+                                       0 0 0 0 0 0 0 0
+                                       0 0 0 0 1 1 1 0)"},
+
+};
+
+// generates an Order Polytope from an instance file containing the adjacency matrix
```

Comment:
In the comment you say that the function takes a file but it takes a string, am I missing something?

Reply:
Initially, my idea was to have a directory containing all the instances as txt files, but later changed to using an unordered_map. Fixed the comment 👍🏽